### PR TITLE
Add POST for users

### PR DESCRIPTION
**Pull request by infiniwave**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -6,18 +6,35 @@ const app = express();
...
 });
 
 app.get('/', (req, res) => {
-  res.send(200);
+    res.send(200);
 });
 
-app.get("/users", async (req, res) => {
-  let users = await User.find({});
+app.get("/users/", async (req, res) => {
```
Comment (stuyy) [inline app.js]: Please fix this. Do not change things that don't need to be changed. Remove the /

**Comment by hypewiththecapitalh** [comment]
This task has already been submitted.

**Comment by infiniwave** [comment]
removed unnecessary /

**Comment by stuyy** [comment]
You need to merge the master branches changes with yours before we can approve and merge

**Pull request closed without merging**